lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -109,6 +109,7 @@
     private final List<SessionPropertyAccessControlRule> sessionPropertyRules;
     private final List<FunctionAccessControlRule> functionRules;
     private final Set<AnySchemaPermissionsRule> anySchemaPermissionsRules;
+    private final List<AuthorizationRule> authorizationRules;
```

Comment:
I'm not sure, but consider adding this to `filter*`.  Generally, if you have any permission on a entity (or contained entity), the entity is visible